src/org/zaproxy/zap/extension/domxss/TestDomXSS.java

```diff
@@ -486,22 +496,17 @@ public void scan() {
 			attackVectors.add(ATTACK_STRINGS[i]);
 		}
 		
-		ArrayList<WebDriverWrapper> drivers = new ArrayList<WebDriverWrapper>();
-		
-		WebDriverWrapper fxDriver;
 		try {
-			fxDriver = this.getFirefoxDriver();
+			ffDriver = this.getFirefoxDriver();
 		} catch (Exception e) {
 			getLog().warn("Skipping scanner, failed to start Firefox: " + e.getMessage());
 			getParent().pluginSkipped(this, Constant.messages.getString("domxss.skipped.reason.browsererror"));
 			return;
 		}
 
-		drivers.add(fxDriver);
-
 		try	{
 			for(String attackVector : attackVectors) {
-				if (scan(drivers, attackVector)) {
+				if (scan(ffDriver, attackVector, getBaseMsg().getRequestHeader().getURI().toString())) {
```

Comment:
Missing the `+ attackVector` when passing the URL.

Reply:
That might be the catch, will update in a sec.